src/sentry/post_process_forwarder/post_process_forwarder.py

```diff
@@ -97,7 +97,7 @@ def _build_streaming_consumer(
     ) -> StreamProcessor[KafkaPayload]:
         configure_metrics(MetricsWrapper(metrics.backend, name="eventstream"))
 
-        cluster_name = settings.KAFKA_TOPICS[topic]["cluster"]
+        cluster_name = settings.KAFKA_TOPICS[topic]["cluster"]  # type: ignore
```

Comment:
we should be able to annotate `KAFKA_TOPICS` in the settings file and fix this ignore